js/redux/store.js

```diff
@@ -18,4 +18,4 @@ export function createStore(reducer, initialState) {
   return {
     getState, dispatch, subscribe
   };
-}
\ No newline at end of file
+}
```

Comment:
missed semicolon

Reply:
function declaration doesn't need a semicolon at the end (after curly brace)